### PR TITLE
Yzhbankov/gh 238

### LINKED ISSUE #238: Restrict  user without credential provided
- [x]  improve api with checking and validation
- [x] on UI just show notification about issues with credentials 
- [x]  add  todo for form  with client validation for next task

400  - just show issue with credentials

### PR DESCRIPTION
**Pull request by yzhbankov**
close #238

### CODE REVIEW

File: apps/web-portal/lib/use-cases/auth/AuthLogin.mjs [hunk 1]
```diff
@@ -2,12 +2,12 @@ import {UseCase} from '@zorko-io/util-use-case'
 import {apiToken} from '../utils'
 
 export class AuthLogin extends UseCase {
-  // eslint-disable-next-line no-unused-vars
-  async run(params) {
-    if (params.token) {
-      apiToken.verify(params.token)
-    }
+  static rules = {
```
Comment (nesterone) [inline apps/web-portal/lib/use-cases/auth/AuthLogin.mjs:5]: Could you double check LIVR format for declaration of both (email/password) or (token) payload , I see the code duplication between both UseCases, and only rules are different

**Review by nesterone — CHANGES_REQUESTED** [review]

File: apps/web-portal/lib/use-cases/auth/AuthLogin.mjs (hunk 1 above)
Comment (yzhbankov) [reply apps/web-portal/lib/use-cases/auth/AuthLogin.mjs:5]: I also think about such LIVR rule, but unfortunately, there are no embedded rules that covered this logic. We can create our own but not sure about it.

Comment (nesterone) [reply apps/web-portal/lib/use-cases/auth/AuthLogin.mjs:5]: What about this one https://livr-spec.org/validation-rules/metarules.html#variable-object ?

Comment (yzhbankov) [reply apps/web-portal/lib/use-cases/auth/AuthLogin.mjs:5]: This rule is not working, at least I could not found

Comment (nesterone) [reply apps/web-portal/lib/use-cases/auth/AuthLogin.mjs:5]: Could you add TODO for finding a way to define such rule with LIVR ?

Comment (yzhbankov) [reply apps/web-portal/lib/use-cases/auth/AuthLogin.mjs:5]: add 'or' rule

File: apps/web-portal/lib/use-cases/auth/AuthLogin.mjs [hunk 2]
```diff
@@ -2,10 +2,28 @@ import {UseCase} from '@zorko-io/util-use-case'
...
+        }
+      }]
+    }, 'required']
+  }
+
+
   async run(params) {
-    if (params.token) {
-      apiToken.verify(params.token)
+    // eslint-disable-next-line no-unused-vars
```
Comment (nesterone) [inline apps/web-portal/lib/use-cases/auth/AuthLogin.mjs:22]: Could we avoid that eslint directive ?

File: .github/dependabot.yml [hunk 3]
```diff
@@ -5,7 +5,7 @@
 
 version: 2
 updates:
-  - package-ecosystem: "" # See documentation for possible values
+  - package-ecosystem: "npm" # See documentation for possible values
```
Comment (nesterone) [inline .github/dependabot.yml:8]: yarn ?

**Review by nesterone — CHANGES_REQUESTED** [review]

File: .github/dependabot.yml (hunk 3 above)
Comment (yzhbankov) [reply .github/dependabot.yml:8]: according to the documentation, yarn is marked as npm option

File: apps/web-portal/lib/use-cases/auth/AuthLogin.mjs (hunk 2 above)
Comment (yzhbankov) [reply apps/web-portal/lib/use-cases/auth/AuthLogin.mjs:22]: yes, but then we need to use all the 'token, password, email' or remove the next row
leave this directive temporary before we add logic related to the pwd and email

**Review by nesterone — APPROVED** [review]

**Pull request merged**